tests/testapp/settings.py

```diff
@@ -31,13 +31,13 @@
 # Application definition
 
 INSTALLED_APPS = [
-    "django.contrib.admin",
-    "django.contrib.auth",
-    "django.contrib.contenttypes",
-    "django.contrib.sessions",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    "dc_utils",
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'dc_utils',
```

Comment:
Can you explain this quotation mark change? I think `black` will just change it back?

Reply:
Looks like I missed running black on the tests app - will update